tools/glb-viewer/src/viewer.js

```diff
@@ -7,7 +7,8 @@ var Viewer = function (canvas) {
     // create the application
     var app = new pc.Application(canvas, {
         mouse: new pc.Mouse(canvas),
-        touch: new pc.TouchDevice(canvas)
+        touch: new pc.TouchDevice(canvas),
+        miniStats: true
```

Comment:
Why is this `miniStats` property needed?

Reply:
It's no longer needed, removed. Also renamed glb-viewer -> viewer.